packages/components/src/tooltip/index.tsx

```diff
@@ -109,7 +109,10 @@ function UnforwardedTooltip(
 	// the tooltip anchor anymore since 0.4.0, so we need to add it manually.
```

Comment:
We might want to expand the comment to explain under which circumstances we're adding it.